ush/ICpert_scripts/config.sh.GEFS_member

```diff
@@ -0,0 +1,55 @@
+MACHINE="hera"
+ACCOUNT="fv3lam"
+EXPT_SUBDIR="${GEFS_expt_subdir}"
+
+USE_CRON_TO_RELAUNCH="TRUE"
+CRON_RELAUNCH_INTVL_MNTS="01"
+
+COMPILER="intel"
+VERBOSE="TRUE"
+
+RUN_ENVIR="community"
+PREEXISTING_DIR_METHOD="rename"
+
+DATE_FIRST_CYCL="20220430"
+DATE_LAST_CYCL="${date_last_cycl}"
+CYCL_HRS=( "00" )
+
+FCST_LEN_HRS="${fcst_len_hrs}"
+LBC_SPEC_INTVL_HRS="6"
+
+PREDEF_GRID_NAME="${predef_grid_name}"
+QUILTING="TRUE"
+HALO_BLEND="20"
+
+CCPP_PHYS_SUITE="FV3_HRRR"
+
+EXTRN_MDL_NAME_ICS="FV3GFS"
+EXTRN_MDL_NAME_LBCS="FV3GFS"
+
```

Comment:
Are these set to FV3GFS here simply because we can't pull GEFS data with the SRW App?  I thought maybe we could do that at this point, but maybe not with this version of the App.

Reply:
I wondered about that but didn't want to explore it for lack of time and we know that this works with @willmayfield 's scripts.  If that does work in this version, we could just set the ICs one to GEFS, but I haven't explored this.

Reply:
OK, no problem.  It works as is!